projects/ui/src/Components/Apis/gloo-gateway-components/ApisTab/ApisList.tsx

```diff
@@ -29,31 +35,35 @@ export function ApisList({
     if (!apiProductsList?.length) {
       return [];
     }
-    return apiProductsList
-      .filter((api) => {
-        let passesNameFilter =
-          !nameFilter ||
-          api.name.toLocaleLowerCase().includes(nameFilter.toLocaleLowerCase());
-        const passesFilterList =
-          !allFilters.length ||
-          allFilters.every(
-            (filter) =>
-              (filter.type === FilterType.name &&
-                api.name
-                  .toLocaleLowerCase()
-                  .includes(filter.displayName.toLocaleLowerCase())) ||
-              (filter.type === FilterType.keyValuePair &&
-                !!api.apiProductMetadata &&
-                api.apiProductMetadata[
-                  parsePairString(filter.displayName).pairKey
-                ] === parsePairString(filter.displayName).value) ||
-              filter.type === FilterType.apiType
-          );
-        return passesNameFilter && passesFilterList;
-      })
-      .sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
just confirming; removing this is on purpose?

Reply:
Yes, I noticed that it messed up the original ordering of `"(0) ..."`, `"(1) ...` in the API Products name in the mock data.

But now I'm thinking about it, there's a chance it could cause issues if the backend doesn't return the API's in a consistent order, so I'll revert it.

Reply:
Sorting is re-enabled now.